### DISCUSSION TITLE
Buffalo LS-WSX Mini

### DISCUSSION THREAD

**Discussion by Lutz24146** [christgau/wsdd#93] [Ideas]
I use wsdd in my Windows 10 workgroup on 3 Buffalo LS-WSXL headless NAS with Debian-Buster 5.10.13, with the architecture "armel".
For updates, can you have a folder for integrate binary-armel in your repository? Then I might not be able to miss updates via apt updates ...
Your script works without errors and thanks for all the work. 
Best regards, lutz

**Comment by christgau** [comment]
It's nice to hear that wsdd also runs on ARMel. I'd like to know which version you are using in your environment? If you don't know, because it is an older version that does not print any version information, you may try the latest release. If it works as well, that's all I'd like to know.

> For updates, can you have a folder for integrate binary-armel in your repository?

Not sure if I get this right, but neither wsdd ships any binaries nor does any package I am aware of. Wsdd is a Python script only. In addition, I do not maintain any packages for Debian. @fxrb is thankfully doing this in his "community" (as I may call it here) package repo. I have no idea what needs to be done to support "armel" but I guess it might be a keyword or so that needs to be added. Let's see if he picks up this topic.

**Reply by Lutz24146** [reply]
Thanks for the feedback,
I was guided by the link:
[https://github.com/christgau/wsdd](url)
I let myself be guided by the link and entered the source list: 
deb https://pkg.ltec.ch/public/ Buster main
I imported the key accordingly   (https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key). 
apt then reports that there is no installation candidate for wsdd for my architecture (armel) .
Since there is no directory for ARMeL at https://pkg.ltec.ch/public/, apt reported this to me logically.
Then I downloaded the package wsdd_0.6.4_all.deb with wget and installed it with dkpi. The script ran immediately in version 0.6.4 and the access time to mine
Buffalo's has been reduced to "immediately" after booting. I think that for an installation on armel in the directory
https://pkg.ltec.ch/public/dists/buster/main/ only a folder for "binary-armel" is missing, which contains the data equivalent to
https://pkg.ltec.ch/public/dists/buster/main/binary-arm64/Packages included. That's all.

But, it works so perfectly too!  Thanks for your understanding..., Lutz

**Reply by christgau** [reply]
Great. Good to know it's the latest version that works for you. 

> I think that for an installation on armel in the directory
https://pkg.ltec.ch/public/dists/buster/main/ only a folder for "binary-armel" is missing, which contains the data equivalent to
https://pkg.ltec.ch/public/dists/buster/main/binary-arm64/Packages included. That's all.

Similar things had been done for #19, IMHO.

**Comment by fxrb** [comment]
It's not as simple as adding a directory or some files ;-).
As _wsdd_ is a python script installing directly from a package file is perfectly fine as mentioned in https://github.com/christgau/wsdd/issues/19#issuecomment-640194513.
Never mind: I have added the `armel` architecture for the Debian distributions `stretch` and `buster`.
